rails/locales/sq.yml

```diff
@@ -0,0 +1,136 @@
+en:
+  activerecord:
+    attributes:
+      user:
+        confirmation_sent_at: Konfirmimi është dërguar më
+        confirmation_token: Tokeni i konfirmimit
+        confirmed_at: Konfirmuar më
+        created_at: Krijuar më
+        current_passëord: Fjalëkalimi i tanishëm
```

Comment:
Not sure why every w is ë now.

Reply:
Hey Jason. W is not an Albanian letter. And since we use English keyboards, we lack ë on them. So, we write w instead of ë and we then replace ë for w.

Reply:
I got it. I also replaced w with ë in keys. I fixed that, too. I'm sorry for the inconvenience.